src/SDKs/CognitiveServices/dataPlane/Language/Language/Language/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Language Client Library")]
 [assembly: AssemblyDescription("Provides API functions for consuming Microsoft Azure Cognitive Services Language APIs.")]
 
-[assembly: AssemblyVersion("1.0.1.0")]
-[assembly: AssemblyFileVersion("1.0.1.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```

Comment:
AssemblyVersion stays the same, AssemblyFileVersion needs to be updated

Reply:
Just want to confirm, AssemblyVersion only changes on breaking changes?